internal/network/command.go

```diff
@@ -304,8 +312,8 @@ func autocompleteNetworks(client *ccloudv2.Client, environmentId string) []strin
 	return suggestions
 }
 
-func getNetworks(client *ccloudv2.Client, environmentId string) ([]networkingv1.NetworkingV1Network, error) {
-	return client.ListNetworks(environmentId)
+func getNetworks(client *ccloudv2.Client, environmentId string, name, cloud, region, cidr, phase, connectionType []string) ([]networkingv1.NetworkingV1Network, error) {
```

Comment:
Why does this need to be a function? It just calls another function with the same signature

Reply:
True. Actually all the networking commands do the same thing, can address this in a separate PR